Quandl.Excel.Console/Version.cs

```diff
@@ -62,9 +61,9 @@ private bool IsExcel32BitInstall(string version)
                     return Introspection.GetPEFormat(folder) == Introspection.PEFormat.PE32;
                 }
             }
-            catch (Exception e)
+            catch (Exception)
             {
-                Utilities.LogToSentry(e);
```

Comment:
I create the catch all and log function here: https://github.com/quandl/quandl-excel-windows/pull/9/files#diff-d24adddba7dd745b35c4d6213367b5d5R28, the errors for installation will be log into a system temp folder which don't have any write permission issue.